calendar/Module.md

```diff
@@ -0,0 +1,102 @@
+## Overview
+The module provides the capability to manage events and calendar operations and supports service account authorization that can provide delegated domain-wide access to GSuite domain and support admin to do operations on behalf of the domain users.
```

Comment:
Shall we rephrase this to: The Google Calendar Connector provides the capability to manage events and calendar operations. It also provides the capability to support service account authorization that can provide delegated domain-wide access to the GSuite domain and support admins to do operations on behalf of the domain users.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884